src/lib.rs

```diff
@@ -282,6 +283,13 @@ impl Pty {
           libc::CLOSE_RANGE_CLOEXEC as c_int,
         );
 
+        if let Some(apparmor_profile) = &opts.apparmor_profile {
```

Comment:
should we ensure linux only via `[cfg(target_os = "linux")]` or we are fine with the runtime error ?

Reply:
let's ensure linux!